src/main/java/me/zetastormy/akropolis/util/TextUtil.java

```diff
@@ -28,28 +29,36 @@
 public class TextUtil {
     private static final MiniMessage MINI_MESSAGE = MiniMessage.miniMessage();
 
+    private static boolean miniplaceholders = false;
```

Comment:
I did this to make it fit your codestyle in PlaceholderUtil.java.

I would suggest introducing static booleans like this to HooksManager.java instead